src/main/resources/kilt.accesswidener

```diff
@@ -1,13 +1,4 @@
 accessWidener v2 named
-
-# Kilt's own access wideners.
-accessible class net/minecraft/server/network/ServerLoginPacketListenerImpl$State
-
-accessible method net/minecraft/client/renderer/texture/SpriteContents$AnimatedTexture getFrameX (I)I
-accessible method net/minecraft/client/renderer/texture/SpriteContents$AnimatedTexture getFrameY (I)I
-transitive-accessible method net/minecraft/world/entity/Entity setRemoved (Lnet/minecraft/world/entity/Entity$RemovalReason;)V
-transitive-extendable method net/minecraft/world/entity/Entity setRemoved (Lnet/minecraft/world/entity/Entity$RemovalReason;)V
-
```

Comment:
Should probably have these automatically added by the TransformerToWidener task, as these are vital AWs.

Reply:
That probably wasn't intentional, will look into it